### PR TITLE
:zap: Validate token creation form

### LINKED ISSUE #955: Add validation to CreateToken form inputs
# ℹ️ Overview
In `CreateToken.tsx`  in the onSubmit function, ensure the `formData` inputs are valid. Throw an error message and prevent submission if they are not valid. 

Requirements: 
   - tokenName - string with no spaces
   - unitName - string with no spaces
   - totalSupply - whole number 
   - decimals - whole number between 1-10
   - assetURL - valid url string
   - clawbackAddr - valid algorand wallet string || null
   - reserveAddr - valid algorand wallet string || null
   - managerAddr - valid algorand wallet string || null
   - freezeAddr - valid algorand wallet string || null

### PR DESCRIPTION
**Pull request by treasurechic**
# ℹ Overview
closes #955 

### 🔐 Acceptance:

- [ ] `yarn test` passes

### CODE REVIEW

File: components/LaunchPad/Token/CreateToken.tsx [hunk 1]
```diff
@@ -90,22 +93,133 @@ export const CreateToken = () => {
...
+    }
+  }
+
+  const createToken = () => {
     setLoading(true)
+    const payload = { ...formData, decimals: Number(decimals), totalSupply: Number(totalSupply) }
+    delete payload.showClawbackAddr
+    delete payload.showFreezeAddr
+    delete payload.showManagerAddr
+    delete payload.showReserveAddr
```
Comment (ericsharma) [inline components/LaunchPad/Token/CreateToken.tsx:223]: I do not understand why you are deleting these properties from the payload. Can you explain your thought process?

**Review by ericsharma — CHANGES_REQUESTED** [review]
Please review my pending comment related to deleting properties from the payload. Also you did not fix the merge conflict and your branch is failing to compile now. In the future, if you are going to try to resolve merge conflicts, make sure that the changes work locally before committing them to the PR.

File: components/LaunchPad/Token/CreateToken.tsx (hunk 1 above)
Comment (treasurechic) [reply components/LaunchPad/Token/CreateToken.tsx:223]: These are not part of what is required to create a token, so I deleted them from the payload object. I am only using them for the UI toggle.

Comment (ericsharma) [reply components/LaunchPad/Token/CreateToken.tsx:223]: Ahh I see my mistake, I misread it. I thought you were deleting the actual properties not the boolean used for the UI toggle.

Comment (treasurechic) [reply components/LaunchPad/Token/CreateToken.tsx:223]: It is not compulsory, you can take it out; the values you need to create a token are inside the formData.

**Review by ericsharma — APPROVED** [review]

**Pull request merged**